packages/editor/src/components/editor-canvas/style.scss

```diff
@@ -0,0 +1,5 @@
+.block-editor__container {
```

Comment:
I'm not certain where this class come from, can't we just use a dedicated class that we pass in `editor-canvas` instead like `editor-canvas__iframe` for something?